### PR TITLE
Add team member nominations policy document

### PR DESCRIPTION
**Pull request by joshuaboniface**

### CODE REVIEW

File: policies-and-procedures/new-team-member-nominations.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Jellyfin's contributor team is a dynamic, ever growing set of contributors who assist in the running of the project. Since Jellyfin is a "do-ocracy", team members make up the backbone of the project, doing numerous jobs such as writing code, reviewing suggestions, moderating the community, and helping to troubleshoot and test.
+
+This document outlines the steps that should be taken to nominate and accept a new member into the team.
+
+## Process
+
+1. Anyone who is currently on the Jellyfin team (defined as a member of #jellyfin-support-team and/or the GitHub organization) may nominate a community member for inclusion in the team.
+
+1. Nominations should reflect the following 4 goals:
```
Comment (MrTimscampi) [inline policies-and-procedures/new-team-member-nominations.md:11]: Might want to fix the numbers for each of these, I'm not sure if Github will auto-correct them.

Comment (joshuaboniface) [reply policies-and-procedures/new-team-member-nominations.md:11]: This is a markdown thing, it does. Having `1., 1.` will auto-format.

**Review by anthonylavado — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]
Used a markdown linter on the document and everything is fine (assuming line-length is not an issue) 👍

File: policies-and-procedures/new-team-member-nominations.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  a) Nominees should demonstrate a consistent, high-quality contribution to the project, e.g. in supporting users, contributing good code, or making & improving clients. While a specific threshold of contributions is not always possible, the nominee should have displayed at least several notable acts, and at least a few successfully merged pull requests or compatable activity in other places.
```
Comment (ThibaultNocchi) [inline policies-and-procedures/new-team-member-nominations.md:13]:
```suggestion
  a) Nominees should demonstrate a consistent, high-quality contribution to the project, e.g. in supporting users, contributing good code, or making & improving clients. While a specific threshold of contributions is not always possible, the nominee should have displayed at least several notable acts, and at least a few successfully merged pull requests or compatible activity in other places.
```

Comment (anthonylavado) [reply policies-and-procedures/new-team-member-nominations.md:13]: Maybe he meant "comparable"?

Comment (ThibaultNocchi) [reply policies-and-procedures/new-team-member-nominations.md:13]: Yeah that's way more believable

**Review by MrTimscampi — APPROVED** [review]

**Review by ferferga — APPROVED** [review]

File: policies-and-procedures/new-team-member-nominations.md (hunk 2 above)
Comment (joshuaboniface) [reply policies-and-procedures/new-team-member-nominations.md:13]: Yes, I did :) Good catch.

**Comment by joshuaboniface** [comment]
> Used a markdown linter on the document and everything is fine (assuming line-length is not an issue) +1

I write so much Markdown that at this point I'm a human Markdown linter. :rofl:

**Pull request merged**